src/vm/mod.rs

```diff
@@ -354,12 +354,9 @@ fn eval_all(
 
 /* Run provided program in a brand new environment, with a transient, empty
  *  database.
- *
- *  Only used by CLI.
```

Comment:
Do we still want this comment above `execute`

Reply:
Oh, yes, we still want this comment -- should be changed to `Only used by CLI and unit tests`

Reply:
done